paywall/data-iframe.1.0.webpack.config.js

```diff
@@ -39,9 +39,19 @@ module.exports = () => {
     output: {
       path: path.resolve(__dirname, 'src', 'static'),
       filename: 'data-iframe.1.0.min.js',
-      globalObject: 'self',
```

Comment:
noticed this just now: this is only needed in a service worker script, and I had copy/pasted from the config I made for service worker. Removing since these scripts actually run in window context